pkg/actions/commands.go

```diff
@@ -15,14 +15,28 @@ import (
 	"crypto/tls"
 	"net/http"
 	"os"
+	"runtime"
 
 	"github.com/eclipse/codewind-installer/pkg/appconstants"
 	"github.com/eclipse/codewind-installer/pkg/errors"
 	logr "github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
-var tempFilePath = "codewind-docker-compose.yaml"
+// Get home directory
+func getHomeDir() string {
```

Comment:
Pulling this out to a function is good but there's about 4 other places we get the homeDir now. Could we make it a global function/variable to make sure we do this the same way everywhere?

Reply:
Update, alternatively if/when we are on Go 12 could we just use: https://golang.org/pkg/os/#UserHomeDir ?

Reply:
Sadly not on go 12 (YET!), hence why we are doing this, but i agree. Will move this into a global func